ksqldb-parser/src/main/java/io/confluent/ksql/parser/KsqlParser.java

```diff
@@ -61,6 +64,15 @@ public static ParsedStatement of(
       return new ParsedStatement(statementText, statement);
     }
 
+    /**
+     * Use masked statement for logging and other output places it could be read by human. It
```

Comment:
Same as above for adding a javadoc on `getStatementText()` and for updating `toString()`. Also for PreparedStatement below.